corehq/apps/app_manager/xpath.py

```diff
@@ -218,6 +218,16 @@ def case(self, instance_name='casedb', case_name='case'):
         quoted = CaseTypeXpath("'{}'".format(self))
         return super(CaseTypeXpath, quoted).case(instance_name, case_name)
 
+    def cases(self, additional_types=[], instance_name='casedb', case_name='case'):
+        quoted = CaseTypeXpath("'{}'".format(self))
+        selector = "{sel}={self}".format(sel=self.selector, self=quoted)
```

Comment:
Nitpick: reusing `self` as a kwarg is a little confusing, maybe just use `quoted` as in the code below?

Reply:
Sure